apps/wallet/src/ui/app/components/accounts/EditableAccountName.tsx

```diff
@@ -67,6 +60,7 @@ export function EditableAccountName({ accountID, name }: { accountID: string; na
 						inputRef.current = e;
```

Comment:
seems like this is the way to do it - https://github.com/react-hook-form/react-hook-form/issues/365